pages/community.tsx

```diff
@@ -0,0 +1,397 @@
+import {
+  AspectRatio,
+  Box,
+  Center,
+  Container,
+  DarkMode,
+  Flex,
+  Icon,
+  Image,
+  SimpleGrid,
+} from "@chakra-ui/react";
+import { CommunityCard } from "components/community/Card";
+import { HomepageFooter } from "components/footer/Footer";
+import { HomepageTopNav } from "components/product-pages/common/Topnav";
+import { useTrack } from "hooks/analytics/useTrack";
+import { NextSeo } from "next-seo";
+import { PageId } from "page-id";
+import { FiZap } from "react-icons/fi";
+import { Heading, LinkButton, Text } from "tw-components";
+import { ThirdwebNextPage } from "utils/types";
+
+const TRACKING_CATEGORY = "community";
+
+const SEO = {
+  title: "Join the Best Web3 Community | Learn, Build, Grow",
+  description:
+    "Join the thirdweb community and share your web3 journey alongside 35,000+ other builders. Learn, build, & grow with thirdweb.",
+};
+
+const CommunitySections = [
+  {
+    title1: "Learn",
+    title2: "web3 development",
+    description:
+      "Whether you're interested in web3 or thirdweb, our mission is to cultivate a community of like-minded individuals who share our passion for the future of decentralized technology.",
+
+    cards: [
+      {
+        image: "",
+        link: "/learn",
+        title: "thirdweb learn",
+        description: "",
+      },
+      // {
+      //   image: "",
+      //   link: "/learn",
+      //   title: "Bootcamp",
+      // description: ""
+      // },
+      {
+        image: "",
+        link: "/events",
+        title: "Events",
+        description: "",
+      },
+      {
+        image: "",
+        link: "/events",
+        title: "Office Hours",
+        description: "",
+      },
+    ],
+  },
+  {
+    title1: "Build",
+    title2: "innovative projects",
+    description:
+      "From hackathons to bounties, open source to community projects, there are so many ways to build no matter your level of contribution. ",
+    cards: [
+      {
+        image: "",
+        link: "/events",
+        title: "Hackathons",
```

Comment:
is it events or hackathons?

Reply:
they are both on the same page for now- we need a hackathon page that is in the works